pkg/kubelet/kubelet.go

```diff
@@ -878,21 +879,21 @@ func (kl *Kubelet) killContainerByID(ID, name string) error {
 }
 
 const (
-	networkContainerName  = "net"
-	NetworkContainerImage = "kubernetes/pause:latest"
+	podInfraContainerName  = "POD" // caps are not allowed by users, but are OK for docker.
```

Comment:
Made that change. Let me know if you prefer to extract it out to some other package.